core/cache/.gitignore

```diff
@@ -1,4 +0,0 @@
-# ignore everything
```

Comment:
Please restore this file; you may have accidentally removed it with force removing the cache, but it should stay there.